editor/create_dialog.cpp

```diff
@@ -785,7 +790,7 @@ CreateDialog::CreateDialog() {
 	vbc->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 	hsc->add_child(vbc);
 
-	search_box = memnew(LineEdit);
+	search_box = memnew(FilterLineEdit);
 	search_box->set_clear_button_enabled(true);
 	search_box->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 	search_box->connect(SceneStringName(text_changed), callable_mp(this, &CreateDialog::_text_changed));
```

Comment:
I think you forgot to remove the line below, which is now in the `FilterLineEdit`.
`search_box->connect(SceneStringName(gui_input), callable_mp(this, &CreateDialog::_sbox_input));`

Reply:
`_sbox_input()` has some custom code for collapsing items, so the signal is still needed. I just forgot to remove the part handled by FilterLineEdit; fixed now.